projectDocs/translating/crowdin.md

```diff
@@ -0,0 +1,100 @@
+# Translating using Crowdin
+
+Crowdin is used to translate the main NVDA interface.
+NVDA's Crowdin project: <https://crowdin.com/project/nvda>.
+
+This document covers setting up a Crowdin account, connecting it with PoEdit, and translating the main interface using Crowdin and PoEdit.
+
+## Setup
+
+### Create Crowdin account
+
+1. Create a [Crowdin account](https://accounts.crowdin.com/register?continue=%2Fproject%2Fnvda).
+1. Message the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org> to request being added as a translator.
+Please include your Crowdin username and the languages you wish to translate.
+
+#### Translation reviews
+
+Translated strings will need to be reviewed and approved by a proofreader before being included in NVDA.

```

Comment:
If I am not mistaken, the job of the approver is not covered in this document. I think that you cannot synchronize poedit with Crowdin to approve strings; the only thing that you can do is to upload your .po file and check auto-approval when doing this operation. To be checked though.

But explaining how strings should be approved is required, especially for people who start translating a new language. Else, this document is useless for new translators.

Reply:
I've added more information here